### PR TITLE
meson.eclass: include pkg-config in the cross file

### PR DESCRIPTION
**Pull request by dm0-**
This fixes cross-compiling current systemd.

### CODE REVIEW

**Comment by monsieurp** [comment]
Thanks @dm0-. Eclass patches follow a different route and are to be sent to the gentoo-dev mailing. Please get someone to review your patch there first and reopen this PR then.

File: eclass/meson.eclass [hunk 1]
```diff
@@ -125,6 +125,7 @@ _meson_create_cross_file() {
 	ar = '${AR}'
 	c = '${CC}'
 	cpp = '${CXX}'
+	pkgconfig = '${PKG_CONFIG}'
```
Comment (floppym) [inline eclass/meson.eclass]: This doesn't appear to be documented anywhere. Are you sure meson even looks at this setting?

Comment (dm0-) [reply eclass/meson.eclass]: I recall looking into meson's source to find the correct variable name to use.  I can look around again for better insight.  Here is an example cross-file using it in the upstream meson repo: https://github.com/mesonbuild/meson/blob/master/cross/ubuntu-mingw.txt#L10 .  At any rate, this change does allow systemd to be configured.

Comment (floppym) [reply eclass/meson.eclass]: Ah, I see it does.

https://github.com/mesonbuild/meson/blob/master/mesonbuild/dependencies/base.py#L183

**Pull request closed without merging**